go/libraries/doltcore/sqle/index/prolly_fields.go

```diff
@@ -129,6 +130,111 @@ func GetField(ctx context.Context, td val.TupleDesc, i int, tup val.Tuple, ns tr
 	return v, err
 }
 
+// Serialize writes an interface{} to the ith field of the Tuple being built.
+func Serialize(ctx context.Context, ns tree.NodeStore, t val.Type, v interface{}) []byte {
```

Comment:
Is it possible to use `PutField` or refactor it so it's usable instead of copy'ing pasting it to this new function that returns `[]byte`?

Reply:
Unfortunately, not really.  PutField does enough other things that there was no clean way to separate out just the serialization logic, unless PutField calls Serialize and then switches on the type _again_. It was simpler to just do this.